crates/types/src/config/rocksdb.rs

```diff
@@ -108,8 +110,16 @@ impl RocksDbOptions {
     }
 
     pub fn rocksdb_write_buffer_size(&self) -> NonZeroUsize {
-        self.rocksdb_write_buffer_size
-            .unwrap_or(NonZeroUsize::new(50_000_000).unwrap()) // 50MB
+        // Value is calculated from other defaults of the system
+        self.rocksdb_write_buffer_size.unwrap_or_else(|| {
+            let common_opts = CommonOptions::default();
+            let all_memtables = common_opts.rocksdb_total_memtables_size();
+            let num_partitions = WorkerOptions::default().bootstrap_num_partitions();
+            // assuming 1 memtables per partition
+            // assuming 256MB for bifrost
+            let buffer_size = (all_memtables - 256_000_000) / (num_partitions * 2) as usize;
```

Comment:
Are those number already in line with the partition processor storage and log store configuration? It looks as if the partition processor store still configures 3 max write buffers. The log store seems to configure 3 max write buffers for the metadata column. The data cf seems to use the default rocksdb write buffer size.

Is the comment assuming 1 memtables per partition in line with `num_partitions * 2` in order to give it some margin of error or because there can always be an additional memtable for each partition being flushed?

Reply:
Ugh, not all the changes were pushed. I'll fix that.

I'll add a couple of comments as a trail for the reasoning as well.